dev/Materials/Acrylic/AcrylicBrush_19h1_themeresources.xaml

```diff
@@ -71,6 +91,8 @@
             <SolidColorBrush x:Key="AcrylicBackgroundFillColorDefaultInverseBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
             <SolidColorBrush x:Key="AcrylicBackgroundFillColorBaseBrush" Color="{ThemeResource SystemColorWindowColor}"/>
             <SolidColorBrush x:Key="SystemControlDefaultBrighteningBrush" Color="{ThemeResource SystemColorWindowColor}"/>
+            <SolidColorBrush x:Key="AccentAcrylicBackgroundFillColorDefaultBrush" Color="{ThemeResource SystemAccentColor}"/>
+            <SolidColorBrush x:Key="AccentAcrylicBackgroundFillColorBaseBrush" Color="{ThemeResource SystemAccentColor}"/>
```

Comment:
`<SolidColorBrush x:Key="AccentAcrylicBackgroundFillColorDefaultBrush" Color="{ThemeResource SystemColorWindowColor}"/>
            <SolidColorBrush x:Key="AccentAcrylicBackgroundFillColorBaseBrush" Color="{ThemeResource SystemColorWindowColor}"/>`

Reply:
Are those the correct high contrast colors to fallback to? @YuliKl can you weigh in?

Reply:
Thanks for checking! Yes, the values @marksfoster listed for high contrast are correct. We actually don't want to show acrylic in HC at all.

Reply:
Alright, thanks for the clarification. Updated the HC brushes now.